tdrop

```diff
@@ -43,6 +43,8 @@ options:
 			for width or height to values relative to the size of the
 			current monitor and force resizing of the dropdown when
 			the monitor changes (default: false)
+	-t		use mouse pointer location for detecting which screen is the current
+			one
```

Comment:
Could you also add a section in the manpage? Mention that this only has an effect when using `-m` / `--monitor-aware`.